YanZhylavy/HW5/Practical task 5.2.py

```diff
@@ -0,0 +1,11 @@
+elements = int(input('''Please, tell how many elements
```

Comment:
good job, 
please look at this
n = int(input('enter some number from fibonacci list= '))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])
print(f'there are some numbers from fibanacci till that you entered {n},\n {fibonacci_numbers}')

Reply:
Thank you, fixed!